test/extended/coreos/coreos.go

```diff
@@ -28,6 +28,15 @@ var _ = g.Describe("[sig-coreos] [Conformance] CoreOS bootimages", func() {
 	defer g.GinkgoRecover()
 
 	oc := exutil.NewCLIWithoutNamespace("coreos")
+
+	g.BeforeEach(func() {
+		microShift, err := exutil.IsMicroShiftDeployment(oc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		if microShift {
+			g.Skip("Skipped in MicroShift")
+		}
+	})
+
 	g.It("TestBootimagesPresent", func() {
```

Comment:
Are these present in hypershift?  I don't know what they are, but if every other form-factor of OCP has these, why shouldn't microshift?

Reply:
These are the RHCOS images for the node and where to get them depending on the platform. Microshift does not run on RHCOS and does not have machine config operator either, so this test may be skipped.